From 578dd56caca5ef302e0a80192a129d05611e45cc Mon Sep 17 00:00:00 2001
From: Jens Reidel <adrian@mainlining.org>
Date: Tue, 2 Sep 2025 02:31:19 +0200
Subject: [PATCH 076/128] dts: sm7150-xiaomi-common: cd-gpio is active low

Untested, but schematic says SD_CARD_DET_N so it should be active low.
U-Boot also requires cd-inverted without this, I'm not sure how SD card
detection in Linux ever worked without this.

Signed-off-by: Jens Reidel <adrian@mainlining.org>
---
 arch/arm64/boot/dts/qcom/sm7150-xiaomi-common.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sm7150-xiaomi-common.dtsi b/arch/arm64/boot/dts/qcom/sm7150-xiaomi-common.dtsi
index f8cc80b946cc..14db8f024211 100644
--- a/arch/arm64/boot/dts/qcom/sm7150-xiaomi-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sm7150-xiaomi-common.dtsi
@@ -669,7 +669,7 @@ &remoteproc_mpss {
 };
 
 &sdhc_2 {
-	cd-gpios = <&tlmm 69 GPIO_ACTIVE_HIGH>;
+	cd-gpios = <&tlmm 69 GPIO_ACTIVE_LOW>;
 
 	vmmc-supply = <&vreg_l9c_3p1>;
 	vqmmc-supply = <&vreg_l6c_3p0>;
-- 
2.43.0

